var/spack/repos/builtin/packages/oommf/package.py

```diff
@@ -0,0 +1,330 @@
+        # This environment variable (OOMMF_ROOT) seems not to be
+        # set at this point, so we have to set it manually for the test:
+        oommfdir = self.get_oommf_path(self.prefix)
+        test_env["OOMMF_ROOT"] = oommfdir
+
+        print("Testing oommf.tcl +version")
+
+        # where is tcl?
+        tclsh = Executable(join_path(spec["tcl"].prefix.bin, "tclsh"))
+        # where is oommf.tcl?
+        oommf_tcl_path = join_path(spec.prefix.bin, "oommf.tcl")
+        # put the command together and execute
+        output = tclsh(
+            oommf_tcl_path, "+version", output=str.split, error=str.split, env=test_env
+        )
+
+        print("output received fromm oommf is %s" % output)
+
+    @run_after("install")
+    def check_install_platform(self):
+        spec = self.spec
+        test_env = {}
+        # OOMMF needs paths to execute
+        test_env["PATH"] = os.environ["PATH"]
+        print("PATH=", test_env["PATH"])
+
+        # Make sure the correct config is found
+        # This environment variable (OOMMF_ROOT) seems not to be
+        # set at this point, so we have to set it manually for the test:
+        oommfdir = self.get_oommf_path(self.prefix)
+        test_env["OOMMF_ROOT"] = oommfdir
+
+        print("Testing oommf.tcl +platform")
+
+        # where is tcl?
+        tclsh = Executable(join_path(spec["tcl"].prefix.bin, "tclsh"))
+        # where is oommf.tcl?
+        oommf_tcl_path = join_path(spec.prefix.bin, "oommf.tcl")
+        # put the command together and execute
+        output = tclsh(
+            oommf_tcl_path, "+platform", output=str.split, error=str.split, env=test_env
+        )
+
+        print("output received fromm oommf is %s" % output)
+
+    @run_after("install")
+    def check_install_stdprob3(self):
+        spec = self.spec
+        test_env = {}
+        # OOMMF needs paths to execute
+        test_env["PATH"] = os.environ["PATH"]
+        print("PATH=", test_env["PATH"])
+
+        # Make sure the correct config is found
+        # This environment variable (OOMMF_ROOT) seems not to be
+        # set at this point, so we have to set it manually for the test:
+        oommfdir = self.get_oommf_path(self.prefix)
+        test_env["OOMMF_ROOT"] = oommfdir
+
+        print("Testing oommf.tcl standard problem 3")
+
+        # where is tcl?
+        tclsh = Executable(join_path(spec["tcl"].prefix.bin, "tclsh"))
+        # where is oommf.tcl?
+        oommf_tcl_path = join_path(spec.prefix.bin, "oommf.tcl")
+        # put the command together and execute
+        oommf_examples = join_path(spec.prefix.usr.bin, "oommf/app/oxs/examples")
+        task = join_path(oommf_examples, "stdprob3.mif")
+        output = tclsh(
+            oommf_tcl_path,
+            "boxsi",
+            "+fg",
+            task,
+            "-exitondone",
+            "1",
+            output=str.split,
+            error=str.split,
+            env=test_env,
+        )
```

Comment:
For me, this looks more readable:
```suggestion
        output = tclsh(oommf_tcl_path, "boxsi", "+fg", task, "-exitondone", "1",
            output=str.split, error=str.split, env=test_env)
```

Reply:
I ran this through black. If you don't mind, I'll leave it as is so that I can keep doing that for future updates.